integration_test/third_party_apps_test/applications/apache/debian_ubuntu/install

```diff
@@ -2,6 +2,18 @@ set -e
 
 sudo apt update
 sudo apt install -y apache2
-(cd /etc/apache2/conf-available/ && sudo curl -O https://raw.githubusercontent.com/Stackdriver/stackdriver-agent-service-configs/master/etc/httpd/conf.d/status.conf)
+sudo mkdir -p /etc/httpd/conf.d/
```

Comment:
this directory changed, and now the tests are failing with: 

main_test.go:1032: Non-retryable error: representative metric [workload.googleapis.com/apache.requests](https://www.google.com/url?q=http://workload.googleapis.com/apache.requests&sa=D) not found, skipping remaining metrics

https://btx.cloud.google.com/invocations/9d46d190-f129-4c1e-94f0-54957484bbf9/targets/logs;config=default/tests

probably want to put it back to /etc/apache2/conf-available?

Reply:
Those test failures are unrelated to this PR. We see the same failures in the nightly tests, and they are caused by metric descriptor inconsistencies.

Reply:
ok, so I'm still left wondering why the config directory changed here.

Reply:
ping on this

Reply:
You're right — the tests would have caught this if they weren't failing for unrelated reasons. Good catch. Fixed.